### ISSUE TITLE
site is not usable and options

### ISSUE THREAD

**Issue by karlcow** [webcompat/webcompat.com#3066]
When selecting site is not usable.

<img width="1101" alt="Capture d’écran 2019-10-29 à 09 27 34" src="https://user-images.githubusercontent.com/505230/67727880-76cb8600-fa2e-11e9-99d0-f9af533eb24d.png">

The proposed choices are not compelling. (in this case this is a scrolling issue). So I guess I need to do something else, but I was hesitant to do it.

**Comment by miketaylr** [comment]
I guess the issue here is that not all possible options are covered, and if it doesn't match, you have to go back to Something else, right?

**Comment by karlcow** [comment]
yes my hesitation came that I thought I was already on the next step and I was not sure I could select the other options. :)
